specification/elastic/resource-manager/Microsoft.Elastic/preview/2024-01-01-preview/elastic.json

```diff
@@ -2573,9 +2573,8 @@
       "type": "object",
       "properties": {
         "openAIResourceId": {
```

Comment:
>"openAIResourceId": { [](http://example.com/codeflow?start=8&length=21)

It says resource ID but it is actually the name? What has been the behaviour so far? Are there customers using this?

Reply:
It has been changed in old API versions too, seems like a breaking change in existing behaviour

Reply:
Approving as the ARM ID validation added was not intended and it is causing failures now. Please change property name to reflect name instead of ID in future API versions.